app/models/manageiq/providers/redhat/infra_manager/refresh/parse/strategies/api4.rb

```diff
@@ -1,4 +1,41 @@
 module ManageIQ::Providers::Redhat::InfraManager::Refresh::Parse::Strategies
   class Api4 < ManageIQ::Providers::Redhat::InfraManager::Refresh::Parse::Parser
+    def self.cluster_inv_to_hashes(inv)
+      result = []
+      result_uids = {}
+      result_res_pools = []
+      return result, result_uids, result_res_pools if inv.nil?
+
+      inv.each do |data|
+        mor = data.id
+
+        # Create a default Resource Pool for the cluster
+        default_res_pool = {
+          :name         => "Default for Cluster #{data.name}",
+          :uid_ems      => "#{mor}_respool",
```

Comment:
Ca we use  `#{cluster.id}_respool` and stop using the `mor` local variable? I guess it means _managed object reference_, but it takes time to come to that conclusion, and may well be wrong.